src/overlays/actors/ovl_En_Po_Composer/z_en_po_composer.c

```diff
@@ -15,20 +15,24 @@ void EnPoComposer_Destroy(Actor* thisx, PlayState* play);
 void EnPoComposer_Update(Actor* thisx, PlayState* play);
 void EnPoComposer_Draw(Actor* thisx, PlayState* play);
 
-void func_80BC5294(EnPoComposer* this, PlayState* play);
-void func_80BC52F0(EnPoComposer* this, PlayState* play);
-void func_80BC53A0(EnPoComposer* this, PlayState* play);
-void func_80BC5404(EnPoComposer* this, PlayState* play);
-void func_80BC5570(EnPoComposer* this, PlayState* play);
-void func_80BC55E0(EnPoComposer* this, PlayState* play);
-void func_80BC5670(EnPoComposer* this, PlayState* play);
-void func_80BC56D0(EnPoComposer* this, PlayState* play);
-void func_80BC5780(EnPoComposer* this, PlayState* play);
-void func_80BC5830(EnPoComposer* this, PlayState* play);
-void func_80BC58E0(EnPoComposer* this, PlayState* play);
-
-#if 0
-ActorInit En_Po_Composer_InitVars = {
+void EnPoComposer_AwaitPlayer(EnPoComposer* this, PlayState* play);
+void EnPoComposer_StartCutscene(EnPoComposer* this, PlayState* play);
+void EnPoComposer_StartedCutscene(EnPoComposer* this, PlayState* play);
+void EnPoComposer_PlayCurse(EnPoComposer* this, PlayState* play);
+void EnPoComposer_IdleStill(EnPoComposer* this, PlayState* play);
+void EnPoComposer_Appear(EnPoComposer* this, PlayState* play);
+void EnPoComposer_Idle(EnPoComposer* this, PlayState* play);
+void EnPoComposer_RaiseArms(EnPoComposer* this, PlayState* play);
+void EnPoComposer_LowerArms(EnPoComposer* this, PlayState* play);
+void EnPoComposer_CutscenePlayCurse(EnPoComposer* this, PlayState* play);
+void EnPoComposer_Roll(EnPoComposer* this, PlayState* play);
+
+void EnPoComposer_SetupAwaitPlayer(EnPoComposer* this);
+void EnPoComposer_SetupStartCutscene(EnPoComposer* this);
+void EnPoComposer_SetupStartedCutscene(EnPoComposer* this);
+void EnPoComposer_StepLightAlpha(EnPoComposer* this);
+
+const ActorInit En_Po_Composer_InitVars = {
```

Comment:
Could you remove the `const` here?
```suggestion
ActorInit En_Po_Composer_InitVars = {
```